components/form/Input.js

```diff
@@ -20,23 +20,34 @@ const Input = forwardRef(
       }
     };
 
+    const inputClassName =
+      type === "color"
+        ? classNames(
+            disabled
+              ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
+              : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0",
+            "border-2 transition-all duration-250 ease-linear rounded px-1 mb-2 block w-[100px]"
+          )
+        : classNames(
+            disabled
+              ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
+              : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0",
+            "border-2 transition-all duration-250 ease-linear rounded px-6 py-2 mb-2 block w-full"
+          );
```

Comment:
This seems difficult to read, is it possible to convert it to something like a `switch` statement?

Reply:
Okay, lets give it a try

Reply:
![change](https://github.com/EddieHubCommunity/BioDrop/assets/87894780/77ce7f95-2c2f-4f84-92f2-14693107adc9) 
 We can rewrite it using a switch statement like this

Reply:
@AbhishekBhure please use the inline comment function of GitHub rather than using a screenshot of the code.

```
switch (type) ...
```

Reply:
Am using this for the first time, Please let me know did I added the comment properly.